src/meta/src/manager/sink_coordination/coordinator_worker.rs

```diff
@@ -252,10 +255,12 @@ impl CoordinatorWorker {
         &mut self,
         mut coordinator: impl SinkCommitCoordinator,
     ) -> anyhow::Result<()> {
-        coordinator.init().await?;
+        let log_store_rewind_start_epoch = coordinator.init().await?;
         loop {
-            let (handle_id, vnode_bitmap, epoch, metadata) =
-                self.handle_manager.next_commit_request().await?;
+            let (handle_id, vnode_bitmap, epoch, metadata) = self
+                .handle_manager
+                .next_commit_request(log_store_rewind_start_epoch)
+                .await?;
```

Comment:
The logic here only works for decoupling sink. For coupling sink with online scaling, we should make sure they won't enter this loop. In other words, we should ban scaling on coupling sinks.